yt_dlp/YoutubeDL.py

```diff
@@ -1056,6 +1056,7 @@ def prepare_outtmpl(self, outtmpl, info_dict, sanitize=None):
             (?P<maths>(?:{math_op}{math_field})*)
             (?:>(?P<strf_format>.+?))?
             (?P<alternate>(?<!\\),[^|)]+)?
+            (?:\&(?P<replacement>.+?))?
```

Comment:
shouldn't this be `.*?` like `default`?
And the above line should be `[^|&)]`

Reply:
also, `&` shouldn't be escaped

Reply:
I thought whether empty replacement values make any sense. But you are right, empty replacement values may be useful in some cases, so using `.*?` here, too.